### DISCUSSION TITLE
Issue with Branding Logo Image Upload on Arcana Dashboard

### DISCUSSION THREAD

**Discussion by dev-saltmines** [arcana-network/discussions#28] [Arcana Dashboard]
I recently attempted to upload my branding logo image to the Arcana Dashboard. However, when I tried to access the image, I encountered the following error message:

`{
    "message": "Not Found"
}`

Could you please assist me in resolving this issue? Here are the steps I followed:

1. Uploaded the branding logo image via the Arcana Dashboard.
2.  Attempted to access the image using the provided link.

Despite following the usual process, the image link returned a "Not Found" message. I would appreciate your guidance on how to fix this and ensure that my branding logo is correctly displayed. (I using testnet version)

**Comment by shaloo** [comment]
Thx for reporting, we will get back shortly.

**Comment by hemang-pant** [comment]
Thanks for reporting this issue, we have started working on this issue. Will update. once we have the fix ready.
Will continue with further communication on the telegram channel.
